springfox-swagger-ui/src/web/js/csrf.js

```diff
@@ -4,6 +4,8 @@
  * @returns {Promise<void>}
  */
 export default async function patchRequestInterceptor(baseUrl) {
+  if (!window.ui.findCsrfToken) return;
```

Comment:
Are the configuration values not accessed via  window.ui.getConfigs().findCsrfToken ??